doc/source/analytics/log_level.md

```diff
@@ -30,24 +42,30 @@ For example, to set it in each container running with the python wrapper, you wo
       }
   ]
 }
-...
+// ...
 ```
 
 Once this has been set, it's possible to use the log in your wrapper code as follows:
 
-```
+```python
 import logging
 
 log = logging.getLogger()
-
 log.debug(...)
 ```
 
-### Setting log level in the Seldon Engine
+## Log level in the service orchestrator
 
-In order to set the log level in the SeldonEngine this can be done by providing the env option to the svcOrchSpec, as follows:
+.. Note:: 
+   When using the Go implementation, setting the ``SELDON_LOG_LEVEL`` to
+   ``WARNING`` and above in the service orchestrator will also change the
+   structure of the log messages, which will be logged as JSON, and will enable
```

Comment:
I am a bit confused about the format. So for one level it is unformatted text and for some it is `JSON`?

Reply:
In Go, because we are following K8s' convetion, you've only got two levels: `0` and `1`. `0` is information that they user must see and `1` is considered as debug.

By default, when you disable `devMode` in the logger, it will only log `0`-level messages, structure them as JSON (to be easier to parse), and it will turn on sampling.

I agree that it's a bit confusing. Maybe we can separate log behaviour as 2 flags:

- `--debug` which controls the log's structure and sampling
- `--log-level` which only controls the verbosity levels

Reply:
Yes, I believe indeed that having these two decoupled makes more sense and provides more flexibility to the user.

Reply:
@RafalSkolasinski we know have both separate. Would be great to hear your thoughts!